src/crimson/os/futurized_store.h

```diff
@@ -60,6 +66,50 @@ class FuturizedStore {
     uint64_t offset,
     size_t len,
     uint32_t op_flags = 0) = 0;
```

Comment:
See above, who is the caller of FuturizedStore::read() ?

Reply:
osdmeta will call store->read.

Reply:
see above